### PR TITLE
added home/video listing page

### PR DESCRIPTION
**Pull request by anupkgurung**
Added home page/video listing page.
Added sidebar option

Home

### CODE REVIEW

File: src/components/VideoCard/VideoCard.jsx [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+                        <img className="w-100" src={thumnail} alt={alt} />
+                    </div>
+                  
+                    <div className="">
+                       {title}
+                    </div>
+                    
+                        <div className="footer-icons">
+                            <span className="material-icons">visibility</span>{creator}
+                        </div>
```
Comment (hrishikeshmane) [inline src/components/VideoCard/VideoCard.jsx]: can be well formatted by prettier

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            const data = await axios.get("/api/videos")
+            console.log(data.data.videos)
+            setVideoList(data.data.videos)
+        }catch(error){
+            showToast("error", "Error occured on fetching videos");
+        }
+    }
+    useEffect(()=>{
+        getAllVideos();
+    },[])
```
Comment (hrishikeshmane) [inline src/pages/Home/Home.jsx:22]: add getAllVideos in dependency list

**Review by hrishikeshmane — APPROVED** [review]
looks good.
Suggestion- Use eslint and prettier in VSCode

File: src/components/VideoCard/VideoCard.jsx [hunk 3]
```diff
@@ -1,3 +1,21 @@
-export const VideoCard = () => {
-    
+
+import "./VideoCard.css"
+export const VideoCard = ({id,title,video,thumnail,creator,alt}) => {
+    return (
+        <>
+            <div className='w-20' id={id}>
```
Comment (ishitakeshawani) [inline src/components/VideoCard/VideoCard.jsx]: add key here

File: src/pages/Home/Home.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import { useState, useEffect } from "react";
+import "../style.css";
+
+export const Home = ()=>{
+    const { showToast } = useToast();
+    const [videoList, setVideoList] = useState([])
+    const getAllVideos = async () => {
+        try{
+            const data = await axios.get("/api/videos")
+            console.log(data.data.videos)
```
Comment (ishitakeshawani) [inline src/pages/Home/Home.jsx:14]: remove console log

File: src/pages/style.css [hunk 5]
```diff
@@ -158,10 +158,16 @@
     height: 2rem;
 }
 .chip {
-    background-color: #efefef;
+    /* background-color: #efefef; */
```
Comment (ishitakeshawani) [inline src/pages/style.css]: remove commented code

File: src/pages/VideoList/VideoList.jsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+import "../style.css";
+
+export const Home = ()=>{
+    return (
+        <>  
+        <div className="flex-row">
+            <Sidebar />
+            <div className="flex-column-center">
+                <div class="chip">
+                    <span class="chip-item">All</span>
```
Comment (ishitakeshawani) [inline src/pages/VideoList/VideoList.jsx:11]: change class to className

**Review by ishitakeshawani — APPROVED** [review]
Added  suggestions
code is readable
just format files by using prettier

**Pull request merged**